_posts/2018-07-24-post-io-meetup.md

```diff
@@ -0,0 +1,25 @@
+
+![Meetup about to begin!]({{page.assets}}/post_io.jpg){:class="img-responsive" : .center-image :height="400px" width="500px"}
+
+Finally the day came, meetup began in the rocking way! So, when two co-oragnizers, [Sneh Pandya](https://twitter.com/SnehPandya18) & [Manan Jadhav](https://twitter.com/curosmj), reached San Francisco, they head their way to [Mountain View](https://en.wikipedia.org/wiki/Mountain_View,_California) where global head-quarter of Google, [Googleplex](https://en.wikipedia.org/wiki/Googleplex) & [Shoreline Amphitheatre](https://en.wikipedia.org/wiki/Shoreline_Amphitheatre) are situated. Shoreline Amphitheatre is place where every year Google IO conference take place. During IO, Google converts whole amphitheatre & it's parking area into IO Venue.
+
+ Countdown for Google IO conference began. Google IO is a three days event. At the first day in morning, they did breakfast at IO venue which was little bit strange for them but they told it was so good. After that, it was time for [keynote](https://www.theverge.com/2018/5/8/17320962/google-io-keynote-2018-announcements-news-android-p). They enjoyed amazing keynote lead by Sundar Pichai. Keynote continued for approximately two hours by different rocking speakers, speaking on upcoming innovative stuffs. It was fantasic experience for them. According to them, if an IO attendee could miss keynote & goes to back stage, he/she can definitely interact with Sundar Pichai & other speakers. After keynote, sessions on different technical stuffs started in the [codelabs](https://events.google.com/io/schedule/?section=may-8). Codelabs are small buildings where sessions on different tech-stuffs take place. There were codelabs for Android, Firebase, Web , Google Accessibility, Android & IoT, Machine Learning, Flutter and many more. Attendees can attend codelabs depending on their specific field of interest or they can even explore different stuffs. After one hour of codelabs, there was [Developer Keynote](https://events.google.com/io/schedule/?section=may-8), where speakers explained innovations in different development frameworks used for developing different products. This keynote too was a great experience for the co-organizers, by which developers can code efficient products. Keynote continued for one hour & again sessions in the codelabs began. And that codelab sessions were in motion on the second & third day of IO, for the whole day, starting from morning to evening. According to co-organizers, codelab can be a best thing to explore & learn different stuffs.
```

Comment:
Please break this into two paragraphs. Read "Updating PRs" sections of the contributing guide to see how to make updates to a PR.